### DISCUSSION TITLE
Have you considered using Renovate to automate dependencies updates?

### DISCUSSION THREAD

**Discussion by lmilbaum** [npalm/terraform-aws-gitlab-runner#919] [Q&A]
Renovate is a very active open source project which is widely used.

**Comment by kayman-mk** [comment]
I know Renovate and I am using it in other projects. Dependabot has some strange behaviours (at least the version we use here), i.e. creates multiple PRs for the same Terraform provider update in different directories, changes the PR message with something like "Dependabot" is rebasing this PR, ...

Would be nice to get a PR from you (@lmilbaum) to activate Renovate here.

**Reply by lmilbaum** [reply]
The Renovate GitHub app should be activated.
Dependabot should be disabled.

**Reply by lmilbaum** [reply]
Bumping. The repo maintainer can onboard the renovate GH app.

**Comment by kayman-mk** [comment]
Read somewhere that the App has been suspended. Don't remember where it was.

Just noticed that I didn't activate a plan. Let's see if it creates some PRs now.

**Reply by lmilbaum** [reply]
You can notice the app is activated by an onboarding PR landing.  Have you activated the app?

**Reply by kayman-mk** [reply]
Yeah, still waiting for the onboarding PR. Should be created automatically, right?

**Reply by lmilbaum** [reply]
Right

**Reply by lmilbaum** [reply]
The onboarding PR was due. Something is off.

**Reply by kayman-mk** [reply]
Can you fix it?

**Reply by lmilbaum** [reply]
Yes. If you can you temporary elevate my permissions to onboard the app.

**Reply by kayman-mk** [reply]
Not possible, sorry. But the App is already installed for the organization.

**Reply by lmilbaum** [reply]
Then check which repositories are enabled

**Reply by lmilbaum** [reply]
Or we can hop on a short call and I can troubleshoot with you

**Comment by kayman-mk** [comment]
I guess I know what the problem was: I had to sign in to developer.mend.io and activate everything. Done now and waiting for the PR.

**Reply by kayman-mk** [reply]
Activated

**Reply by lmilbaum** [reply]
yay. I'll contribute configuration updates.
